paymentsheet/src/test/java/com/stripe/android/paymentsheet/viewmodels/PrimaryButtonUiStateMapperTest.kt

```diff
@@ -386,6 +388,29 @@ class PrimaryButtonUiStateMapperTest {
         }
     }
 
+    @Test
+    fun `screen buy button override should be used complete flow when available`() = runTest {
```

Comment:
Do we already have tests for the previous case? Maybe another test to ensure the other possible outcomes of this test?